lib/matplotlib/traitlets.py

```diff
@@ -0,0 +1,231 @@
+                raise TraitError(("""a trait getter method must
+                                   have 2 or fewer arguments"""))
+            return meth(*args)
+        else:
+            self.__base_get__(obj,cls)
+
+    @property
+    def __base_set__(self):
+        return super(GetSetTraitType,self).__set__
+
+    def __set__(self, obj, value):
+        if hasattr(obj, '_'+self.name+'_setter'):
+            meth = getattr(obj, '_'+self.name+'_setter')
+            if not callable(meth):
+                raise TraitError(("""a trait setter method
+                                   must be callable"""))
+            argspec = getargspec(meth)
+            if isinstance(meth, MethodType):
+                argspec -= 1
+            if argspec==0:
+                args = ()
+            elif argspec==1:
+                args = (self,)
+            elif argspec==2:
+                args = (self, value)
+            else:
+                raise TraitError(("""a trait setter method must
+                                   have 2 or fewer arguments"""))
+            value = meth(*args)
+        self.__base_set__(obj, value)
+
+
+
+class TransformInstance(TraitType):
+
+    info_text = ('a Transform instance or have an'
+                 ' `_as_mpl_transform` method')
+
+    def validate(self, obj, value):
+        if isinstance(value, Transform):
+            return value
+        elif hasattr(value, '_as_mpl_transform'):
+            conv = value._as_mpl_transform(self.axes)
+            return self._validate(conv)
+        trait.error(obj, value)
+
+class GSTransformInstance(TransformInstance,GetSetTraitType): pass
+
+
+
+class Callable(TraitType):
+    """A trait which is callable.
+
+    Notes
+    -----
+    Classes are callable, as are instances
+    with a __call__() method."""
+
+    info_text = 'a callable'
+
+    def validate(self, obj, value):
+        if callable(value):
+            return value
+        else:
+            self.error(obj, value)
+
+class ColorType(TraitType):
+    """A trait representing a color, can be either in RGB, or RGBA format.
+
+    Arguments:
+        force_rgb: bool: Force the return in RGB format instead of RGB. Default: False
+        as_hex: bool: Return the hex value instead. Default: False
+        default_alpha: float (0.0-1.0) or integer (0-255) default alpha value.
+
+    Accepts:
+        string: a valid hex  color string (i.e. #FFFFFF). 7 chars
+        tuple: a tuple of ints (0-255), or tuple of floats (0.0-1.0)
+        float: A gray shade (0-1)
+        integer: A gray shade (0-255)
+
+    Defaults: RGBA tuple, color black (0.0, 0.0, 0.0, 0.0)
+
+    Return:
+       A hex color string, a rgb or a rgba tuple. Defaults to rgba. When
+       returning hex string, the alpha property will be ignored. A warning
+       will be emitted if alpha information is passed different then 0.0
+
+    """
+    metadata = {
+        'force_rgb': False,
+        'as_hex' : False,
+        'default_alpha' : 0.0,
```

Comment:
I think you want this to be 1.0.  Think of "alpha" as "opacity", not "transparency".